tools/KoreBuild.Console/Commands/InstallToolsCommand.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.CommandLineUtils;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace KoreBuild.Console.Commands
+{
+    internal class InstallToolsCommand : SubCommandBase
+    {
+        private string KoreBuildSkipRuntimeInstall => Environment.GetEnvironmentVariable("KOREBUILD_SKIP_RUNTIME_INSTALL");
+        private string PathENV => Environment.GetEnvironmentVariable("PATH");
+        private string DotNetInstallDir => Environment.GetEnvironmentVariable("DOTNET_INSTALL_DIR");
+
+        public override void Configure(CommandLineApplication application)
+        {
+            base.Configure(application);
+        }
+        
+        protected override int Execute()
+        {
+            var installDir = GetDotNetInstallDir();
+
+            Log($"Installing tools to '{installDir}'");
+            
+            if(DotNetInstallDir != null && DotNetInstallDir != installDir)
+            {
+                Log($"installDir = {installDir}");
+                Log($"DOTNET_INSTALL_DIR = {DotNetInstallDir}");
+                Log("The environment variable DOTNET_INSTALL_DIR is deprecated. The recommended alternative is DOTNET_HOME.");
+            }
+
+            var dotnet = GetDotNetExecutable();
+            var dotnetOnPath = GetCommandFromPath("dotnet");
+
+            if (dotnetOnPath != null && (dotnetOnPath != dotnet))
+            {
+                Log($"dotnet found on the system PATH is '{dotnetOnPath}' but KoreBuild will use '{dotnet}'");
+            }
+
+            var pathPrefix = Directory.GetParent(dotnet);
+            if (PathENV.StartsWith($"{pathPrefix};"))
```

Comment:
`StringComparison.OrdinalIgnoreCase`

Also, you'll need to make this x-plat friendly by also using `System.IO.Path.PathSeparator` instead of hard-coding the semicolon.